Mathlib/Analysis/CStarAlgebra/ContinuousMap.lean

```diff
@@ -36,12 +40,16 @@ namespace ContinuousMap
 
 variable [TopologicalSpace α] [CompactSpace α]
 
+#adaptation_note /-- 2025-03-29 for lean4#7717 had to add `norm_mul_self_le` field. -/
 instance [NonUnitalCStarAlgebra A] : NonUnitalCStarAlgebra C(α, A) where
+  norm_mul_self_le := CStarRing.norm_mul_self_le
```

Comment:
@j-loreaux do you understand why this is happening? A lot of `norm_mul_self_le`s are being added manually now. @kmill are you able to add a comment somewhere explaining this fallout? It is one of very few "negative" changes in this PR.

Reply:
Yes, this issue is on my radar. I have a brief comment about it [on Zulip](https://leanprover.zulipchat.com/#narrow/channel/428973-nightly-testing/topic/.2323495.20adaptations.20for.20nightly-2025-03-31/near/509199201).

The problem is that the type of the `CStarRing` parent depends on the fields of `NonUnitalCStarAlgebra`, and that's inhibiting it from using any pre-existing instances. It's unclear what the underlying issue is yet, but I hope there will be a reasonable fix.